CleanWeather/Scenes/CityList/CityListPresenter.swift

```diff
@@ -9,11 +9,29 @@
 import UIKit
 
 protocol CityListPresenter {
```

Comment:
tak jak z kontrollerem, zrob SuperPresenter w ktorym są te 2 metody ze spinnerem i errorem i nie będzie trzeba powtarzac tego w kazdym presenterze

Reply:
Zrobiłem SharedPresenter, ale na razie nie wiem jak przeskoczyć to, że jest tam potrzebny weak var controller: CityListPresentable? (na razie skomentowałem go w CityListPresenter i umieściłem w SharedPresenter) Wiem, że nie tak to powinno być, ale chyba muszę się z tym przespać, albo może mi z rana rzucisz jakąś wskazówkę. :) Pamiętam, że mówiłeś coś o protokołach przy tym temacie, ale nie wiem na razie jak się do tego zabrać.